ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-nightly.yaml

```diff
@@ -212,6 +212,16 @@ tests:
     test:
     - ref: openshift-qe-ingress-perf
     workflow: rosa-aws-sts
+- as: perfscale-rosa-multiaz-network-perf-3nodes-periodic
+  cron: 0 21 * * 0
+  steps:
+    cluster_profile: aws-perfscale-qe
+    env:
```

Comment:
Same comment mentioned for 4.12 nightly applicable here.

Reply:
Ack, will update them!